pandas/tests/groupby/test_groupby_dropna.py

```diff
@@ -162,6 +162,13 @@ def test_groupby_dropna_series_by(dropna, expected):
     tm.assert_series_equal(result, expected)
 
 
+@pytest.mark.parametrize("dropna", (False, True))
+def test_grouper_dropna_propagation(dropna):
+    df = pd.DataFrame({"A": [0, 0, 1, None], "B": [1, 2, 3, None]})
+    gb = df.groupby("A", dropna=dropna)
+    assert gb.grouper.dropna == dropna
```

Comment:
Does this have any user facing impact? The Grouper itself is not a public object

Reply:
> Does this have any user facing impact? The Grouper itself is not a public object

Doubt it. I only added this while working on groupby(dropna=...) stuff.

`Grouper` does have a page in the api ref though so it's at least nominally exposed:
https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Grouper.html

Reply:
@WillAyd maybe you're thinking we should only be testing the public API and not any internals?